plugins/feeds/public/malwaremustdiecncs.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from datetime import datetime, timedelta
+from core.observables import Hostname
+from core.feed import Feed
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+
+class MalwareMustDieCncs(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=24),
+        "name": "MalwareMustDieCncs",
+        "source" : "https://malwared.malwaremustdie.org/rss.php",
+        "description": "List of cncs",
+    }
+
+    def update(self):
+        since_last_run = datetime.now() - self.frequency
+        for item in self.update_xml('item', ["title", "description"]):
+            if datetime.strptime(item["description"], "%d/%b/%Y") < since_last_run:
+                break
```

Comment:
You may want to change `break` for `continue` unless you know for sure that the items are ordered in reverse chronological order.